src/Agent.Worker/Build/FileContainerServer.cs

```diff
@@ -108,7 +110,6 @@ public class FileContainerServer
                         catch
                         {
                             // Fall back to FCS upload if we cannot upload to blob
-                            context.Warn(StringUtil.Loc("BlobStoreUploadWarning"));
```

Comment:
There is a catch here which won't throw the error further to the top try...catch block - do we need to forward it further to print the error?

Reply:
There is no need to throw the error further, the function of this catch block is to process errors and switch to the FCS in case of any errors during uploading to Blob Store. Error is already logged in BlobUploadAsync method. In this catch block, it is just changing the state of _uploadToBlob_ variable that is used to control the choice of service for uploading artifacts (BlobeStore or FCS). If we will throw an error here, falling back to FCS won't happen.

Reply:
Could it make sense to leave warning message here then?

Reply:
This warning still exists, I have just moved its creation to the BlobUploadAsync method because in this particular place I don't have info about the host which we are trying to access (this information is required for our extended logs). I don't think that we need to create a warning for the same error twice.